src/main/java/org/jabref/gui/actions/ActionFactory.java

```diff
@@ -148,13 +147,9 @@ public ButtonBase configureIconButton(Action action, Command command, ButtonBase
         button.getStyleClass().add("icon-button");
 
         // For some reason the graphic is not set correctly, so let's fix this
+        // ToDO: Find a way to reuse JabRefIconView
```

Comment:
So the comments can be removed now?

Reply:
Idk, i just left it there, because I did not change anything in the bigger picture nor put JabRefIconView to some use here. I did not want to delete any reminder one of the earlier developers set.
I work pretty much the same: I write something, I realize, there could be a more elegant approach, but I don't want to loose focus on what I'm writing, so I just leave a ToDo.